dipy/tracking/stopping_criterion.pyx

```diff
@@ -277,19 +279,22 @@ cdef class CmcStoppingCriterion(AnatomicalStoppingCriterion):
             raise RuntimeError("Unexpected interpolation error " +
                                "(exclude_map - code:%i)" % exclude_err)
 
-        rng = np.random.default_rng()
+        # rng = np.random.default_rng()
+
         # test if the tracking continues
         if include_result + exclude_result <= 0:
             return TRACKPOINT
         num = max(0, (1 - include_result - exclude_result))
         den = num + include_result + exclude_result
         p = (num / den) ** self.correction_factor
-        if rng.random() < p:
+        # if rng.random() < p:
```

Comment:
Remove this rather than leaving as a comment?

Reply:
thanks, removed.